src/protocols/ethereum/components/TransferSendForm.vue

```diff
@@ -133,13 +134,11 @@ export default defineComponent({
       maxPriorityFeePerGas,
       updateFeeList,
     } = useEthFeeCalculation();
-    const { getProtocolAvailableTokens } = useFungibleTokens();
-
-    const ethTokensAvailable = computed(() => getProtocolAvailableTokens(PROTOCOLS.ethereum));
+    const { accountAssets } = useAccountAssetsList({ searchTerm: ref('') });
```

Comment:
```suggestion
    const { accountAssets } = useAccountAssetsList();
```
I don't see why we need this